python/src/trezorlib/models.py

```diff
@@ -36,12 +39,17 @@ class TrezorModel:
 
 # ==== internal names ====
 
+USBID_TREZOR_ONE = (0x534C, 0x0001)
+USBID_TREZOR_CORE = (0x1209, 0x53C1)
```

Comment:
don't we have different USBID for T3T1? 🤔

Reply:
Nope